blocks/text/text.css

```diff
@@ -58,6 +58,12 @@ main .block.text.muted {
   padding-bottom: 0.5rem;
 }
 
+main .block.text.extra-muted, main .block.text.extra-muted p {
+  font-size: var(--body-font-size-xs);
```

Comment:
Its agreed to correct the muted version. I already had a card for same, for which PR is raised
https://github.com/hlxsites/sunstar/pull/328

extra-muted is not required.

Reply:
I will stick with the muted variant then